test/autogen/test_agent.py

```diff
@@ -0,0 +1,32 @@
+def test_agent():
+    from flaml.autogen.agent import Agent
+
+    dummy_agent_1 = Agent(name="dummy_agent_1")
+    dummy_agent_2 = Agent(name="dummy_agent_2")
+
+    dummy_agent_1.receive("hello", dummy_agent_2)  # receive a str
+    dummy_agent_1.receive(
+        {
+            "content": "hello",
+        },
+        dummy_agent_2,
+    )  # receive a dict
+    # receive dict with wrong field, there should be no error raised
```

Comment:
Why is it "wrong field"?

Reply:
Changing to "receive dict without openai fields to be printed, such as "content", 'function_call'"